pkg/controllers/securitypolicy_controller.go

```diff
@@ -71,7 +86,7 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			err = errors.New("Security Policy CR cannot be created in System Namespace")
 			log.Error(err, "failed to create security policy CR", "securitypolicy", req.NamespacedName)
 			metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerUpdateFailTotal, METRIC_RES_TYPE)
-			return ctrl.Result{}, err
+			return ctrl.Result{}, nil
```

Comment:
Why update code here?

Reply:
I only revert it back, it's changed based on main branch's change, since it's a normal case, so shouldn't put nil return. otherwise, it will be re queued again.